test/library/standard/Path/victor-ludorum/testcases_isAbsPath.chpl

```diff
@@ -0,0 +1,15 @@
+   use Path;
+
+   writeln(isAbsPath("/foo/bar"));
+
+   writeln(isAbsPath("./baz"));
+	 
+   writeln(isAbsPath("/baz/.."));
+	 
+   writeln(isAbsPath("baz/"));
+	 
+   writeln(isAbsPath("//server"));
+	 
+   writeln(isAbsPath("\\server"));
```

Comment:
Shouldn't this case fail for non-windows environments?

Reply:
@lydia-duncan Yeah mam!! This test case fail for non-windows environment . I have thought that we have to add testcases for windows environment in a separate file . Therefore I have added this testcase file as a separate file .

Reply:
Ooooh, I misread the output.  Sorry!

Reply:
No problem mam 😊 !!